### DISCUSSION TITLE
TS0601 - door sensor is not supported

### DISCUSSION THREAD

**Discussion by pilou8830** [Koenkk/zigbee2mqtt#11997] [Devices]
hello the door sensor (Luminance Door Senosr Model : ZG-102ZL) powered by Tuya
seen by zigbee2mqtt as being a: TS0601 ( _TZE200_pay2byax ) And it is not in the list of compatible devices.
I am under Home Assistant Supervisor with zigbee2mqtt.
I thought I could manually add this door sensor, but it's too much but I can't get out of it. can you help me?

**Comment by pilou8830** [comment]
**thank you** my device has been added to the list of supported devices
only this remains in the zigbee2mqtt log.
But everything seems to be working normally.
![04](https://user-images.githubusercontent.com/48477947/161273581-00c78326-4df1-4cc5-8d63-63227ada59b2.JPG)

**Comment by swobyy** [comment]
Hello, how did you solve it? I have the same problem...
Thank you for your help
Swoby

**Comment by pilou8830** [comment]
I haven't done anything.
it is the one who manages the json files who made the modifications.
if you look here:
https://github.com/Koenkk/zigbee-herdsman-converters/tree/master/devices
the tuya.js file has been modified to add the TS0601 _TZE200_paybyax

{
        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_kltffuzl'}, {modelID: 'TS0601', manufacturerName: '_TZE200_fwoorn8y'},
            {modelID: 'TS0601', manufacturerName: '_TZE200_pay2byax'}],
        model: 'TM001-ZA/TM081',
        vendor: 'TuYa',
        description: 'Door and window sensor',
        fromZigbee: [fz.tm081],
        toZigbee: [],
        exposes: [e.contact(), e.battery()],
    }

you just have to restart the pairing

**Comment by pilou8830** [comment]
hello i had to reinstall everything.
But now I have a problem on zigbee2mqtt (1.18.1).
it does not recognize modelID: 'TS0601', manufacturerName: '_TZE200_pay2byax'. It's a door/window opening sensor.
it is however in the list of Tuya:
![tuya](https://user-images.githubusercontent.com/48477947/162902104-b65dc6f4-96e7-4b24-8223-02ea4ed0b5c1.JPG)
thank you for your help

**Comment by pilou8830** [comment]
I am trying to create a new device running TuYa.
a door opening sensor. It appears as:
  Zigbee model 'TS0601' and manufacturer name '_TZE200_pay2byax'
here is what I did:
created a json file and added 3 lines in Z2M configuration

![fichier json](https://user-images.githubusercontent.com/48477947/163174798-2205c8c8-2171-40a3-ad35-fa525ef7eefd.JPG)
there are syntax errors line 29??

![3 fichiers](https://user-images.githubusercontent.com/48477947/163175096-eeabc5f1-ee53-4e91-a915-46e94b9271df.JPG)

can you correct me already?

**Comment by pilou8830** [comment]
using Zigbee2mqtt Edge (development) my sensor is recognized as in version 1.17.1 edge
a bunch of devices aren't officially in the stable release yet, if I understand correctly.
I would have liked to create it myself but it is too complicated. Especially for the TuYa.
Anyhow, thanks for your help

**Reply by Koenkk** [reply]
You are probably using the wrong repo: 
If you are still on 1.18.1 check first point of https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.19.0 (the latest z2m version is 1.25.0)
